React-Native/TodoList/src/app/screens/Main/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ * @flow
+ * @lint-ignore-every XPLATJSCOPYRIGHT1
+ */
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { View } from 'react-native';
+import Header from './components/Header';
+import Input from './components/Input';
+import List from './components/List';
+import Footer from './components/Footer';
+import todoActions from '../../../redux/Todo/actions';
+import styles from './styles';
+
+class Main extends Component {
+
+  // Chequear esto, todavía no hice nada para que el input me llene ese item que pido
+  addItem = item => {
```

Comment:
i don't think the point in defining this functions... directly pass the functions in props..